### PR TITLE
Update Web Messenger instructions

### PR DESCRIPTION
**Pull request by lemieux**
0 0 0 0-8000-guide-web-messenger- 3

### CODE REVIEW

File: pages/guide/web-messenger.md [hunk 1]
```diff
@@ -144,10 +106,10 @@ You're almost there. Save your Tag and don't forget to publish your changes. You
...
-To install Smooch on your Wordpress site, you need to [download our plugin](https://github.com/smooch/smooch-wordpress) and install it through Wordpress.
+To install Smooch on your Wordpress site, you need to [download our Web Messenger](https://github.com/smooch/smooch-wordpress) and install it through Wordpress.
 
-1. Download the ZIP from our [plugin github page](https://github.com/smooch/smooch-wordpress)
-2. Connect on your Wordpress dashboard and go on the plugin section
+1. Download the ZIP from our [Web Messenger github page](https://github.com/smooch/smooch-wordpress)
+2. Connect on your Wordpress dashboard and go on the Web Messenger section
 3. Click on "Add New" and then "Upload Plugin"
 
 	![Wordpress](/images/wordpress_plugin.png)
```
Comment (mspensieri) [inline pages/guide/web-messenger.md]: Can't comment on the correct line, but app token is still mentioned a few lines below on line 156 before / 118 after

File: pages/guide/web-messenger.md [hunk 2]
```diff
@@ -12,38 +12,44 @@ The Smooch Web Messenger is unique in that it allows you to seamlessly move the
...
+    }(window,document,"Smooch","<app-id>");
+</script>
 ```
 
-#### Step 2: Initialize Smooch with your new app token
 
-Once Smooch has been included on your web page, you're almost done. Simply initialize the plugin using this code snippet
+
+#### Step 2: Initialize Smooch with your new app id
```
Comment (mspensieri) [inline pages/guide/web-messenger.md]: see https://github.com/smooch/smooch-docs/pull/366#discussion_r137006223, "ID" should be capitalized

File: pages/guide/web-messenger.md [hunk 3]
```diff
@@ -54,53 +60,7 @@ In your code:
...
-// or
-
-define(function(require) {
-	var Smooch = require('Smooch');
-	// ...
-});
+Smooch.init({appId: '<app-id>'});
 ```
 
 ### Google Tag Manager
```
Comment (jpjoyal) [inline pages/guide/web-messenger.md]: I was under the impression we we're moving those alternative install instructions to a separate FAQ. Let's use this opportunity to do that and just have a link to those in here (ie " can also include it using npm. We also have install instruction for Google tag manager, Wordpress .... ")

Comment (lemieux) [reply pages/guide/web-messenger.md]: It was just a suggestion, but sure I'll move them over.

**Pull request merged**